### PR TITLE
feat: Stabilize Deno.osRelease() API

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/deno/issues/15928

### CODE REVIEW

File: cli/dts/lib.deno.ns.d.ts [hunk 1]
```diff
@@ -169,6 +169,22 @@ declare namespace Deno {
...
+   * ```
+   *
+   * Requires `allow-env` permission.
+   * Under consideration to possibly move to Deno.build or Deno.versions and if
+   * it should depend sys-info, which may not be desirable.
+   *
+   * @tags allow-env
+   * @category Runtime Environment
+   */
+  export function osRelease(): string;
```
Comment (dsherret) [inline cli/dts/lib.deno.ns.d.ts:186]: This returns `"6.2.9200"` on Windows 10 19043.2006. I probably don't understand windows versioning.

Comment (Uzlopak) [reply cli/dts/lib.deno.ns.d.ts:186]: I think deno uses the wrong windows api. It uses GetVersionEx and not RtlGetVersion. Thats why it returns the wrong version.

**Comment by Uzlopak** [comment]
@dsherret 
#9862

File: cli/dts/lib.deno.ns.d.ts (hunk 1 above)
Comment (bartlomieju) [reply cli/dts/lib.deno.ns.d.ts:186]: Thanks for the heads up, we're gonna stabilize anyway and fix it in a follow up as this API being unstable is a blocker for several Node modules.

**Pull request merged**